### PR TITLE
UI issue fixes

### LINKED ISSUE #17: UI Issues
@JaganKaartik Please work on the following UI issues - 
1. The checkbox for `Contacts` page needs to be center aligned. 

Ref - 
image

For this, please update `neetoUI` package to the latest version and use a `div` wrapper with `flex` properties. 

2. The vertical spacing between the `SubHeader` and that of the listing tables don't match the design.

### PR DESCRIPTION
**Pull request by JaganKaartik**
Fixes #17

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ export default function ContactTable({
 
   return (
     <div className="w-full px-9">
-      <table className="nui-table nui-table--checkbox nui-table-actions">
+      <table className="mt-10 nui-table nui-table--checkbox nui-table-actions">
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx:34]: Please use padding here.

File: app/javascript/src/components/Dashboard/Notes/NoteTable.jsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ export default function NoteTable({
 }) {
   return (
     <div className="w-full px-24">
-      <table className="nui-table nui-table--checkbox nui-table-actions">
+      <table className="mt-10 nui-table nui-table--checkbox nui-table-actions">
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/NoteTable.jsx:12]: Please use padding here.

**Review by karthiknmenon — COMMENTED** [review]
@JaganKaartik Please fix the minor suggestion.

File: app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx (hunk 1 above)
Comment (JaganKaartik) [reply app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx:34]: pt-10 ?

File: app/javascript/src/components/Dashboard/Notes/NoteTable.jsx (hunk 2 above)
Comment (JaganKaartik) [reply app/javascript/src/components/Dashboard/Notes/NoteTable.jsx:12]: pt-10 ?

Comment (karthiknmenon) [reply app/javascript/src/components/Dashboard/Notes/NoteTable.jsx:12]: @JaganKaartik In the design it's 48px, so let's use `pt-12`.

Ref - 
image

**Pull request merged**